rules/windows/process_creation/proc_creation_win_impacket_file_indicators.yml

```
@@ -0,0 +1,26 @@
+title: HackTool - Impacket File Indicators
+id: 03f4ca17-de95-428d-a75a-4ee78b047256
+related:
+    - id: 6e2a900a-ced9-4e4a-a9c2-13e706f9518a
+      type: similar
+status: experimental
+description: Detects file creation events with filename patterns used by Impacket.
+references:
+    - https://thedfirreport.com/2025/05/19/another-confluence-bites-the-dust-falling-to-elpaco-team-ransomware/
+    - https://github.com/fortra/impacket
+author: "The DFIR Report, IrishDeath"
+date: 2025-03-01
```

```suggestion
date: 2025-05-18
```
The date should be same with the date as you create the PR.